site/gatsby-site/src/utils/normalizeRequest.js

```diff
@@ -0,0 +1,14 @@
+// query strings in a function hosted on Netlify behave differently than on Node
+export default function normalizeRequest(req) {
```

Comment:
Any reason not to make this functional? I suppose it's faster to mutate in place...

Reply:
Yeah, I'm not a fan of mutations, but the validator library expects a request, so I didn't want to make things too complicated.